azkaban-web-server/src/main/java/azkaban/webapp/Status.java

```diff
@@ -17,37 +17,9 @@
 
 package azkaban.webapp;
 
-import azkaban.executor.Executor;
-import com.google.common.collect.ImmutableMap;
-import java.util.Map;
-
 /**
- * This POJO is used by GSON library to create a status JSON object. Certain warnings do not apply
- * here.
+ * This is a marker interface for representing azkaban status.
  */
-@SuppressWarnings({"FieldCanBeLocal", "unused"})
-public class Status {
-
-  private final String version;
-  private final String pid;
-  private final String installationPath;
-  private final long usedMemory, xmx;
-  private final boolean isDatabaseUp;
-  private final Map<Integer, Executor> executorStatusMap;
+public interface Status {
```

Comment:
It will be good to have abstract class with common code rather than interface. or a class?